jest.config.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
   moduleNameMapper: {
     "^(\\.\\.?\\/.+)\\.jsx?$": "$1",
   },
+  modulePathIgnorePatterns: ["./weaver/core/identity-management"],
```

Comment:
This module does get exercised in Weaver transaction pipelines, though currently only the `iin-agent` subfolder (other subfolders are placeholders for mechanisms that are specified in the Weaver RFCs but haven't been implemented yet).

Is this module causing any issues that necessitates its exclusion?

Reply:
This file is specific to the jest testing system, and there are no jest tests in weaver.  The weaver convention of copying packages when building causes collisions in the haste map, so jest tests are unable to run.  In my branch where I am building the dependencies as part of running the test, I have had to make additional exceptions.